depends/packages/openssl.mk

```diff
@@ -1,9 +1,9 @@
 # Pretty much taken verbatim from Bitcoin Core
 package=openssl
-$(package)_version=1.0.1m
+$(package)_version=1.0.1k
```

Comment:
I guess you can accept it since I know it works the way it is now and I
think BC Core still uses 1.0.1k.